spec/requests/internal/broadcasts_spec.rb

```diff
@@ -48,6 +48,17 @@
       end
     end
 
+    describe "PUT /internal/broadcasts" do
+      let!(:broadcast) { create(:welcome_broadcast, active: false) }
+
+      it "updates the Broadcasts last_active_at timestamp" do
+        expect do
+          put "/internal/broadcasts/#{broadcast.id}", params: params
+        end.to change { broadcast.reload.active }.from(false).to(true)
+        expect(broadcast.last_active_at).not_to eq(2.days.ago)
```

Comment:
I think you want Timecop here to ensure timing issues don't occur. Also, instead of this being 2 days ago, you should store the old time and assert it is not equal to it. This way if someone changes that factory default this spec's assertion will be updated with it rather than break. 
```
old_time = broadcast.last_active_at
# update
expect(broadcast.last_active_at).not_to eq(old_time)
```

Reply:
Great call @mstruve! I'll make this change now.